src/integrationTest/java/org/opensearch/security/DoNotFailOnForbiddenTests.java

```diff
@@ -408,4 +434,32 @@ public void shouldPerformStatAggregation_negative() throws IOException {
         }
     }
 
+    @Test
+    public void shouldPerformCatIndices_positive() throws IOException {
+        try (RestHighLevelClient restHighLevelClient = cluster.getRestHighLevelClient(LIMITED_USER)) {
+            Request getIndicesRequest = new Request("GET", "/_cat/indices");
+            // High level client doesn't support _cat/_indices API
+            Response getIndicesResponse = restHighLevelClient.getLowLevelClient().performRequest(getIndicesRequest);
+            List<String> indexes = new BufferedReader(new InputStreamReader(getIndicesResponse.getEntity().getContent())).lines()
+                .collect(Collectors.toList());
+
+            assertThat(indexes.size(), equalTo(1));
+            assertThat(indexes.get(0), containsString("marvelous_songs"));
+        }
+    }
+
+    @Test
+    public void testDnfofPermissions_negative() {
+        for (final String permission : disallowedDnfof) {
+            assertThat(DNFOF_PATTERNS.matcher(permission).matches(), equalTo(false));
+        }
+    }
+
+    @Test
+    public void testDnfofPermissions_positive() {
+        for (final String permission : allowedDnfof) {
+            assertThat(DNFOF_PATTERNS.matcher(permission).matches(), equalTo(true));
+        }
+    }
```

Comment:
Both testDnfofPermissions_negative and testDnfofPermissions_positive are spinning up a cluster, because of the @Class rule in this file.  Looks like `PrivilegesEvaluatorUnitTest.java` already exists to handle cases like this, can you move these tests over there?

Reply:
Moved in - [4185872](https://github.com/opensearch-project/security/pull/3236/commits/41858729da8b2af78dd7bde4267613deaf611fd2).